extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1174,11 +1181,31 @@ export class AmpA4A extends AMP.BaseElement {
             'src': srcPath,
             'name': nameData,
           }, SHARED_IFRAME_PROPERTIES));
+      if (method == XORIGIN_MODE.NAMEFRAME) {
+        // TODO(bradfrizzell): change name of function and var
+        const attributes = this.generateSentinelAndContext(iframe);
+        attributes['creative'] = creative;
+        const name = JSON.stringify(attributes);
+        iframe.setAttribute('name', name);
```

Comment:
no, because I need the iframe to already exist to pass in to the generateSentinelAndContext, so I have to wait until it's made